counterpartylib/lib/backend/addrindex.py

```diff
@@ -91,9 +91,20 @@ def make_call(chunk):
             executor.submit(make_call, chunk)
     return list(responses)
 
+
 def extract_addresses(txhash_list):
     logger.debug('extract_addresses, txs: %d' % (len(txhash_list), ))
     tx_hashes_tx = getrawtransaction_batch(txhash_list, verbose=True)
+
+    return extract_addresses_from_txlist(tx_hashes_tx, getrawtransaction_batch)
+
+
+def extract_addresses_from_txlist(tx_hashes_tx, _getrawtransaction_batch):
```

Comment:
I think this is from the https://github.com/rubensayshi/counterparty-lib/pull/10/files patch.